packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1316,3 +1316,6 @@ export const MEETING_PERMISSION_TOKEN_REFRESH_REASON = 'ttl-join';
 
 // constant for named media group type
 export const NAMED_MEDIA_GROUP_TYPE_AUDIO = 1;
+
+// constant for statsAnalyzer report.callingServiceType
+export const CALLING_SERVICE_TYPE = 'LOCUS' as const;
```

Comment:
not sure if the calling service is a right name , isn't locus a signaling server ?

or calling service type indicate what service is sending metrics ?

Reply:
It's specification for MQE requested [here](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-510208)
calls are LOCUS-based and that's where name is from. I can change it to `SIGNALING_SERVICE_TYPE` if you want but later we would still need to do `callingServiceType: SIGNALING_SERVICE_TYPE`